owncloudApp/release/output.json

```diff
@@ -0,0 +1 @@
+[{"outputType":{"type":"APK"},"apkInfo":{"type":"MAIN","splits":[],"versionCode":-1,"enabled":true,"outputFile":"./owncloud_2.9.3-release.apk","fullName":"release","baseName":"release"},"path":"./owncloud_2.9.3-release.apk","properties":{}}]
```

Comment:
What is this used for?

Reply:
this should not be there, it's a additional file to an apk file.

Reply:
deleted